src/helpers/ClauseResultsHelpers.ts

```diff
@@ -102,7 +102,14 @@ export function findAllLocalIdsInStatement(
         aliasMap[v] = statement.expression.localId;
         // Determine the localId for this alias.
         if (statement.localId) {
-          alId = statement.localId;
+          // Older translator versions require with statements to use the statement.expression.localId + 1 as the alias Id
+          // even if the statement already has a localId. There is not a clear mapping for alias with statements in the new
+          // translator, so they will go un highlighted but this will not affect coverage calculation
+          if (statement.type === 'With') {
+            alId = (parseInt(statement.expression.localId, 10) + 1).toString();
```

Comment:
```suggestion
            alId = (parseInt(statement.expression.localId) + 1).toString();
```
Is there a reason we're explicitly specifying base 10 here? Seems to make the code slightly less readable.

Reply:
That's what we did in the code before and do a few lines down

Reply:
Yeah- looks like it was a part of the initial big bang of functionality creation for fqm-execution. *shrug*